scout/adapter/mongo/query.py

```diff
@@ -137,7 +137,14 @@ def build_variant_query(
             case_obj = self.case(display_name=similar_case_display_name, institute_id=institute_id)
             if case_obj:
                 LOG.debug("Search for cases similar to %s", case_obj.get("display_name"))
-                similar_cases = self.get_similar_cases(case_obj)
+
+                hpo_terms = []
+                for term in case_obj.get("phenotype_terms", []):
```

Comment:
A small observation is that all uses of the cases_by_phenotype have some variation of this. Perhaps it should be a conditional part of the function instead? But, nvm if it creates issues, there are like four places where it is used so far...